gammapy/stats/variability.py

```diff
@@ -228,3 +229,65 @@ def compute_flux_doubling(flux, flux_err, coords, axis=0):
     }
 
     return doubling_dict
+
+
+def TimmerAlgorithm(power_spectrum, even_freq=True):
+    """Implementation of the Timmer algorithm to simulate a time series from a power spectrum
+
+    Parameters
+    ----------
+    power_spectrum: `~numpy.ndarray`
+        Array representing the power spectrum used to generate the time series.
+
+    even_freq: bool, optional
+        Parameter needed to distinguish between even and odd number of data points, for handling of the Nyquist frequency.
+        If True, the Nyquist frequency is real by definition. Default is True.
+
+    Returns
+    -------
+    time_series: `~numpy.ndarray`
+        Simulated time series.
+
+    References
+    ----------
+    ..[Timmer1995]"On generating power law noise", J. Timmer and M, Konig, section 3.
+    """
+    random_numbers = np.random.normal(
```

Comment:
You are assuming a time binning of 1.0. Should be configurable, no?

Reply:
Absolutely right, thanks. I can resolve this by changing the rng using `random_state`